weasyl/define.py

```diff
@@ -576,6 +576,20 @@ def convert_date(target=None):
     return result[1:] if result and result[0] == "0" else result
 
 
+def convert_iso_date(target=None):
```

Comment:
Since the optional form isn’t used, let’s keep it non-optional for now.

```suggestion
def convert_iso_date(target):
```